x-pack/test_serverless/api_integration/test_suites/common/platform_security/roles_routes_feature_flag.ts

```diff
@@ -26,15 +26,22 @@ import { FtrProviderContext } from '../../../ftr_provider_context';
 export default function ({ getService }: FtrProviderContext) {
   const platformSecurityUtils = getService('platformSecurityUtils');
   const roleScopedSupertest = getService('roleScopedSupertest');
-  let supertestWithAdminScope: SupertestWithRoleScopeType;
+  let supertestAdminWithApiKey: SupertestWithRoleScopeType;
+  let supertestAdminWithCookieCredentials: SupertestWithRoleScopeType;
   const es = getService('es');
 
   describe('security', function () {
     describe('Roles', () => {
       before(async () => {
-        supertestWithAdminScope = await roleScopedSupertest.getSupertestWithRoleScope('admin', {
+        supertestAdminWithCookieCredentials = await roleScopedSupertest.getSupertestWithRoleScope(
+          'admin',
+          {
+            useCookieHeader: true,
+            withInternalHeaders: true,
+          }
+        );
+        supertestAdminWithApiKey = await roleScopedSupertest.getSupertestWithRoleScope('admin', {
           withInternalHeaders: true,
```

Comment:
Just a note - we'll want to evaluate making some of these endpoints truly public in serverless. Testing here with an API key and internal headers is fine.